.github/workflows/build-docker-images.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: Build docker images
```

Comment:
Nitpicking: I actually like `CI` here, because that's pretty much the gist of it. We run almost all of our CI through bake, so the fact that it builds docker images might be considered an implementation detail :wink: